### PR TITLE
Duration in milliseconds

### PR DESCRIPTION
**Pull request by baudev**
It could be interesting having every duration in milliseconds. 
I see any advantage having duration in seconds except losing precision.

There are some more steps before merging this PR (if only you would):

- [ ] Edit tests
- [ ] Edit documentation

### CODE REVIEW

**Comment by mcollina** [comment]
unit tests are failing, would you mine updating them ad well?

Il giorno mer 20 giu 2018 alle 08:26 baudev <notifications@github.com> ha
scritto:

**Comment by baudev** [comment]
It's what I've planned! But when I'll have time.

File: lib/run.js [hunk 1]
```diff
@@ -245,8 +245,8 @@ function run (opts, cb) {
     }
 
     if (typeof opts.duration === 'string') {
-      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration)
-      else opts.duration = Number(opts.duration.trim())
+      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration) * 1000
```
Comment (thekemkid) [inline lib/run.js]: Would you please change this `timestring(opts.duration) * 1000` to `timestring(opts.duration, 'ms')` if you can, see: https://www.npmjs.com/package/timestring#return-time-value

File: lib/run.js [hunk 2]
```diff
@@ -245,8 +245,8 @@ function run (opts, cb) {
...
+      else opts.duration = Number(opts.duration.trim()) * 1000
```
Comment (thekemkid) [inline lib/run.js]: should this be `* 1000`? surely if duration was a string it should be in ms, too?

**Comment by baudev** [comment]
@thekemkid thanks for this more beautiful way of doing it.

**Comment by mcollina** [comment]
Tests still need updating!

**Comment by mcollina** [comment]
Closing for no activity, feel free to reopen!

**Pull request closed without merging**